### ISSUE TITLE
[accessibility] Source Tabs should be keyboard navigable

### ISSUE THREAD

**Issue by Garbee** [devtools-html/debugger.html#1801]
The source tabs for open files are generic `divs` with no helpers for accessibility. This makes them un-tabbable and pretty difficult to know what is active using a screen reader.

### Solution

My recommended solution is that we use radio inputs. The label would be the tab name, but a `0 x 0` radio input would facilitate tabbing between available tabs as well as the screen readers ability to state which is selected. With this, we'd make the close buttons always active with a good `aria-label` so that tabs can be easily closed using keyboard navigation. This is also one recommended example in the [W3C wiki](https://www.w3.org/wiki/Accessibility_basics#Defining_interaction) for some basic tabs.

Example codepen for the proposed solution: http://codepen.io/Garbee/pen/wgezLL

The codepen doesn't have any state hookup, it's purely about seeing tab-ability (radio is inferred by the number of tabs it takes to move between buttons.)

**Comment by clarkbw** [comment]
I feel like I need to research this more, I don't understand the accessible needs or best practices very well.  I just started reading this: http://simplyaccessible.com/article/danger-aria-tabs/ I'm not sure if that's useful for you as you investigate solutions.

**Comment by Garbee** [comment]
This absolutely needs research and some user testing before rolling out. It isn't something we should change so rapidly.

That article is explaining essentially why you may not want to use the `aria` roles for a tab setup. My proposed solution is to avoid needing that at all, by just using radio inputs with good labels. That way the UA does the heavy lifting for us. For example, we wouldn't be handling keyboard events to detect whether the user wants to switch to that tab. Since, we'd just look for the radio becoming checked and switch contents and refocus on that. It alleviates a bunch of work on our part, however it would still need plenty of testing to make sure people who *rely* on screen readers would be able to navigate it as I expect.

**Comment by jasonLaster** [comment]
I'm moving this to the [enhancements] board. Also added this to #1779. 

I'd like to get to this soon as it's similar to the command bar, but I'm not sure if we have the capacity at the moment.

Also, when we do this we should build the tabs component out of the debugger and pull it in. Or find a good library we can use.

[enhancements]: https://github.com/devtools-html/debugger.html/projects/5
